logstash-core/spec/logstash/output_delegator_spec.rb

```diff
@@ -96,14 +105,18 @@
         end
 
         it "should not utilize the worker queue" do
-          expect(subject.send(:worker_queue)).not_to receive(:pop)
-          subject.multi_receive(events)
+          expect(subject.send(:worker_queue)).to be_nil
```

Comment:
This is expecting there to not be a worker queue defined if we're using a threadsafe worker.